arches_references/src/arches_references/utils.ts

```diff
@@ -247,3 +247,9 @@ export const vFocus = {
         setTimeout(() => el && el.focus({ focusVisible: true }), 5);
     },
 };
+
+export const shouldUseContrast = () => {
+    return Array.from(document.getElementsByTagName("link")).some((el) =>
+        el.href.endsWith("accessibility.css"),
+    );
+};
```

Comment:
I expect we'll replace this with something better once we have infra around consuming system settings.

Reply:
ya I'm wondering if this interaction can also be stubbed out in core at a very basic level as well, since the theme lives there now. We'll definitely use this across applications so if it can proved out in core instead, I think it'll be helpful.

Reply:
https://github.com/archesproject/arches/pull/11327